tests/bearlib/abstractions/LinterTest.py

```diff
@@ -328,6 +347,30 @@ def test_process_output_corrected(self):
                                           original))
         self.assertEqual(len(results), 2)
 
+    def test_process_output_unidiff(self):
+        uut = (linter(sys.executable, output_format='unidiff')
+               (self.EmptyTestLinter)
+               (self.section, None))
+
+        diff = ['--- a/some-file.c', '+++ b/some-file.c', '@@ -1,3 +1,4 @@',
+                '-void main() {', '-       return 09;', '+void main()',
+                '+{', '+       return 9;', ' }']
```

Comment:
could you write it like this? :D
```
diff = ['line 1',
        'line2',
        'line3',
        ...]
```
then it's easier for a human to read the patch :)